lib/logstash/filters/mutate.rb

```diff
@@ -233,6 +245,18 @@ def remove(event)
   end # def remove
 
   private
+  def select(event)
+    replacement = LogStash::Event.new
+
+    @select.each do |field|
+      field = event.sprintf(field)
+      next unless event.include?(field)
+      replacement[field] = event[field]
+    end
+    event.overwrite replacement
```

Comment:
small coding style comment: I would add parenthesis to this call, so it's more clear to read.